Error: Unused lexical variable ‘move-point-in-windows’

Compiling latest code from git (emacs and haskell-mode) as 24/09/2016:

    # make EMACS=/home/aplanas/bin/emacs
    EMACS check-emacs-version
    Using EMACS = /home/aplanas/bin/emacs, version = 25.2.50.1
    mkdir build-25.2.50.1
    EMACS build-25.2.50.1/haskell-menu.elc
    EMACS build-25.2.50.1/haskell-font-lock.elc
    EMACS build-25.2.50.1/haskell-complete-module.elc
    EMACS build-25.2.50.1/haskell-string.elc
    EMACS build-25.2.50.1/haskell-repl.elc
    EMACS build-25.2.50.1/inf-haskell.elc
    EMACS build-25.2.50.1/haskell-session.elc
    EMACS build-25.2.50.1/haskell-sandbox.elc
    EMACS build-25.2.50.1/haskell-process.elc

    In toplevel form:
    haskell-process.el:160:1:Error: Unused lexical variable ‘move-point-in-windows’
    make: *** [Makefile:69: build-25.2.50.1/haskell-process.elc] Error 1

--- a/haskell-lexeme.el
+++ b/haskell-lexeme.el
@@ -138,8 +138,7 @@ When match is successful, match-data will contain:
   (match-text 2) - whole qualified identifier
   (match-text 3) - unqualified part of identifier
   (match-text 4) - closing backtick"
-  (let ((begin (point))
-        (match-data-old (match-data))
+  (let ((match-data-old (match-data))
         first-backtick-start
         last-backtick-start
         qid-start

--- a/haskell-process.el
+++ b/haskell-process.el
@@ -160,9 +160,7 @@ HPTYPE is the result of calling `'haskell-process-type`' function."
 (defun haskell-process-log (msg)
   "Effective append MSG to the process log (if enabled)."
   (when haskell-process-log
-    (let* ((append-to (get-buffer-create "*haskell-process-log*"))
-           (windows (get-buffer-window-list append-to t t))
-           move-point-in-windows)
+    (let* ((append-to (get-buffer-create "*haskell-process-log*")))
       (with-current-buffer append-to
         ;; point should follow insertion so that it stays at the end
         ;; of the buffer

